docs/user/convert/function_usage.rst

```diff
@@ -604,15 +613,6 @@ Example: ``uuidZ3Centroid(linestring('LINESTRING(102 0, 103 1, 104 0, 105 1)', d
 
 See :ref:`customizing_z_index` for details on Z3 intervals.
 
-base64
```

Comment:
Whew...  should we mention that 'base64' is deprecated somehow and indicate that the new function name is 'base64Encode'?